utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -98,6 +103,34 @@ class HtmlParser private constructor(
     return ViewCompat.getLayoutDirection(view)
   }
 
+  /**
+   * Searches for all URLSpans in current text replaces them with our own ClickableSpans
+   * forwards clicks to provided function.
+   */
+  fun TextView.handleUrlClicks(onClicked: ((String) -> Unit)? = null) {
```

Comment:
See my comment in strings.xml--I think we can avoid this by leveraging a callback & custom tag similar to concept card.

Reply:
This is needed as the anchor tags are present in Policies pages that is taken from Web, do I need to replace them with custom tags?

Reply:
Ah I see, shouldn't we use Linkify for that, instead? I.e. https://developer.android.com/reference/android/text/util/Linkify#WEB_URLS.

Reply:
Done.